index.html

```diff
@@ -216,5 +216,82 @@ <h2 class="aboutTitle">Skills</h2>
         </li>
       </ul>
     </section>
+    <section class="contact">
+      <div class="contactHeader">
+        <h2 class="contactTitle">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+      </div>
+
+      <form
+        class="contactForm"
+        action="https://formspree.io/f/mknarzzq"
+        method="post"
+      >
+        <input
+          id="fullName"
+          name="fullName"
+          type="text"
+          placeholder="Full Name"
+          required
+        />
+        <img
+          class="fieldSeparator"
+          src="/images/separatorForm.svg"
+          alt="separator"
+        />
+        <input
+          id="email"
+          name="email"
+          type="email"
+          placeholder="Email address"
+        />
+        <img
+          class="fieldSeparator"
+          src="/images/separatorForm.svg"
+          alt="separator"
+        />
+
+        <textarea
+          id="message"
+          name="message"
+          placeholder="Enter Text Message"
+        ></textarea>
```

Comment:
- [ ] Kindly, update the `textarea` tag to take in a maximum of 500 characters, by making use of the `maxlength` attribute of the `input`s/`textarea` tags, as suggested in the requirement. 😀